packages/concerto-core/test/serializer/objectvalidator.js

```diff
@@ -233,7 +233,7 @@ describe('ObjectValidator', function () {
 
             (function () {
                 objectValidator.visit(concerto.getTypeDeclaration(data), parameters);
-            }).should.throw(/Instance undefined has property previousOwners with type undefined that is not derived from test.Person\[\]/);
+            }).should.throw('Instance "undefined" has a property "previousOwners" with type "undefined" that is not derived from "test.Person[]".');
```

Comment:
I'm wondering if the `undefined` here is a bug? the sentence doesn't read quite right.  Could we double check?